pkg/capabilities/capabilities.go

```diff
@@ -212,6 +212,7 @@ func MustNewCapabilityInfo(
 // TODO: this timeout was largely picked arbitrarily.
 // Consider what a realistic/desirable value should be.
 // See: https://smartcontract-it.atlassian.net/jira/software/c/projects/KS/boards/182
+// TODO: CHECK THIS IS AS EXPECTED ->After this time has passed the call will return - so this is a maximum not a default, caller can choose to cancel the context before this point, but not after, so should be maximumExecuteTimeout, but is this the desired behaviour.
```

Comment:
@ettec This looks correct to me -- we called it a `defaultExecuteTimeout` with a view to making overridable at a later stage. I agree maximum makes more sense, and happy for you to modify the code accordingly 👍